test: skip paths function test

IWANABETHATGUY opened the pull request:
Related to https://github.com/rolldown/rolldown/issues/6737, since the test failed too frequently, disable for now.  
Note: I can't reproduce it locally.

IWANABETHATGUY commented:
* **#6771** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/6771?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/6771?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`
<hr />

## How to use the Graphite Merge Queue

Add the label <kbd>graphite: merge</kbd> to this PR to add it to the merge queue.

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.dev/invite/github/rolldown?ref=merge-queue-instructions-in-stack-comment&prId=6771).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

hyf0 reviewed (approved):
Something wrong with the format

IWANABETHATGUY committed:
test: skip paths function test (#6771)

Related to https://github.com/rolldown/rolldown/issues/6737, since the test failed too frequently, disable for now.
Note: I can't reproduce it locally.

IWANABETHATGUY commented:
> Something wrong with the format

You mean formatting? Yeah, it is weird that dprint did not exit with none zero code, probably test dir is ignored by dprint, I will try to fix it in the next pr.
